app/services/fastpass/FastPassService.scala

```diff
@@ -92,14 +91,24 @@ trait FastPassService extends EventSink with CurrentSchemeStatusHelper {
     } yield ()
   }
 
-  private def autoProgressToSiftOrFSAC(applicationId: String): Future[Unit] = {
+  private def autoProgressToSiftOrFSAC(applicationId: String)(implicit hc: HeaderCarrier): Future[Unit] = {
+    def notifySiftEntered(siftStatus: ProgressStatuses.ProgressStatus): Future[Unit] = {
+      if(siftStatus == ProgressStatuses.SIFT_ENTERED) {
+        applicationSiftService.sendSiftEnteredNotification(applicationId)
+      } else {
+        Future.successful(())
+      }
+    }
+
     val res = for {
       preferences <- schemePreferencesService.find(applicationId)
     } yield {
       val hasSiftableScheme = schemesRepository.siftableSchemeIds.intersect(preferences.schemes).nonEmpty
       if (hasSiftableScheme) {
         val siftStatus = applicationSiftService.progressStatusForSiftStage(preferences.schemes)
-        appRepo.addProgressStatusAndUpdateAppStatus(applicationId, siftStatus)
+        appRepo.addProgressStatusAndUpdateAppStatus(applicationId, siftStatus).flatMap { _ =>
+          notifySiftEntered(siftStatus).map(_ => ())
```

Comment:
This will seem a subtle point but the 'leaf' method, in this case ```notifySiftEntered``` should not have the if condition. Happy to discuss further.